src/pages/search/index.tsx

```diff
@@ -38,20 +39,22 @@ export default function Search() {
   }, [scrollTo]);
 
   return (
-    <HCRCApolloPage
-      uri={ROUTES.SEARCH}
-      className="pageLayout"
-      navigation={<Navigation />}
-      content={
-        <ApolloProvider client={eventsApolloClient}>
-          <SearchPage
-            SearchComponent={AdvancedSearch}
-            pageTitle={'eventSearch.title'}
-          />
-        </ApolloProvider>
-      }
-      footer={<FooterSection />}
-    />
+    <MatomoWrapper>
```

Comment:
Couldn't the MatomoWrapper be used in _app.tsx instead? No need to use it in every page then.

Reply:
should be per nextjs page